zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -204,6 +204,13 @@ public void recordPathStat(Stat stat, OptionalLong notificationTime) {
     }
   }
 
+  @Override
+  public String toString() {
+    StringBuilder stringBuilder = new StringBuilder();
+    stringBuilder.append(_uid);
```

Comment:
I just notice that _uid here was not made final field, could you please fix it? It should be a final field.

Reply:
changed to final